Quick Pad/Dialog/Settings.xaml.cs

```diff
@@ -374,6 +376,46 @@ public async void LoadCrashList()
             NotifyPropertyChanged(nameof(ShowNoCrashCongratz));
         }
 
+        public async void ClearCrashList()
+        {
+            StorageFolder folder = ApplicationData.Current.LocalFolder;
+            var cf = await folder.TryGetItemAsync("Crash") as StorageFolder;
+            if (cf != null)
+            {
+                var files = await cf.GetFilesAsync();
+                foreach (var file in files)
+                    await file.DeleteAsync();
+            }
+            if (AllCrashes is null)
+                AllCrashes = new ObservableCollection<CrashInfo>();
+            else
+                AllCrashes.Clear();
+        }
+
+        public async void SendCrashReports()
+        {
+            EmailMessage emailMessage = new EmailMessage()
+            {
+                Body = "[QuickPad][CrashReport] I want to send these crash report",
+            };
+            string front = "something_???";
+            string provider = "outlook?";
+            emailMessage.To.Add(new EmailRecipient($"{front}@{provider}.com"));
```

Comment:
Still, don't know which email to send to.
Please don't put the email directly or some bot that crawling on the internet might put you on spam list :/

Reply:
I dont need it to send directly, more like an option for the user to manually send if they want help.

Reply:
Maybe just copy a text will do?